### PR TITLE
Add placeholder Apple touch icon and <link> in `index.html`

### PR DESCRIPTION
**Pull request by arthurvr**
Commit in H5BP: https://github.com/h5bp/html5-boilerplate/commit/345925f04384b59929d8d39ba428cef548921408. The icon has the recommended size ([180x180](https://github.com/h5bp/html5-boilerplate/commit/1c4fcf32142e888946db579df0efbb52542e0ed1)).

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -187,6 +187,7 @@ module.exports = yeoman.generators.Base.extend({
       this.mkdir('app/images');
       this.mkdir('app/fonts');
       this.copy('main.js', 'app/scripts/main.js');
+      this.copy('apple-touch-icon.png', 'app/apple-touch-icon.png');
```
Comment (silvenon) [inline app/index.js]: Maybe put this next to `this.copy('favicon.ico', 'app/favicon.ico');`?

Comment (arthurvr) [inline app/index.js]: Done

**Comment by arthurvr** [comment]
I can PR this for generator-webapp too.

**Comment by silvenon** [comment]
That would be awesome :+1: 

Could you ensure white background?

photo 08 03 2015 14 25 26

**Comment by arthurvr** [comment]
@silvenon That was the image I was givin for https://github.com/yeoman/generator-mobile/issues/56. I shall update it but actually wonder if it makes sense. It's a placeholder, you know.

**Comment by arthurvr** [comment]
@silvenon done

**Comment by silvenon** [comment]
It makes sense, it's meant to be on a white background, see?

photo 08 03 2015 15 08 09

Thanks a lot.

**Pull request merged**